api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -90,6 +90,25 @@ message RedisProxy {
     // this limit, then redirection will fail and the original redirection error will be passed
     // downstream unchanged. This limit defaults to 100.
     google.protobuf.UInt32Value max_upstream_unknown_connections = 6;
+
+    // All ReadPolicy settings except MASTER may return stale data because replication is
```

Comment:
Isn't this specific to Redis Cluster? Shouldn't this somehow be part of config that only applies to Redis Cluster?

Reply:
Right now this is only implemented for Redis Cluster, but the replica concept is not limited to Redis Cluster, the stand alone Redis nodes can have replicas (either manually managed or managed through Redis sentinel).
Also the read policy concept is more for the proxy than the cluster.  Not tie a cluster to a specific read policy, it's possible to setup 2 listeners with different read policy for the same cluster.  This is helpful for service that's trying to migrate away from their read your own rights apis.  They can set the old and new api to connect to different ports while they migrate.

Reply:
I guess my point is that from a user perspective it's not clear to me what this actually does and the fact that it only works with Redis Cluster right now. Can you make the docs more clear? I will review the code change. Thank you!